model/src/main/java/org/openremote/model/security/UserResource.java

```diff
@@ -126,62 +116,43 @@ public interface UserResource {
     @Operation(operationId = "resetSecret", summary = "Reset the secret for a user in a realm")
     String resetSecret(@BeanParam RequestParams requestParams, @PathParam("realm") String realm, @PathParam("userId") String userId);
 
-    @GET
-    @Path("{realm}/userRoles/{userId}")
-    @Produces(APPLICATION_JSON)
-    @Operation(operationId = "getUserRoles", summary = "Retrieve client roles for a user in a realm")
-    Role[] getUserRoles(@BeanParam RequestParams requestParams, @PathParam("realm") String realm, @PathParam("userId") String userId);
-
     @GET
     @Path("{realm}/userRoles/{userId}/{clientId}")
     @Produces(APPLICATION_JSON)
     @Operation(operationId = "getUserClientRoles", summary = "Retrieve client roles for a user using a client ID in a realm")
-    Role[] getUserClientRoles(@BeanParam RequestParams requestParams, @PathParam("realm") String realm, @PathParam("userId") String userId, @PathParam("clientId") String clientId);
+    String[] getUserClientRoles(@BeanParam RequestParams requestParams, @PathParam("realm") String realm, @PathParam("userId") String userId, @PathParam("clientId") String clientId);
 
     @GET
     @Path("{realm}/userRealmRoles/{userId}")
     @Produces(APPLICATION_JSON)
     @Operation(operationId = "getUserRealmRoles", summary = "Retrieve realm roles for a user in a realm")
-    Role[] getUserRealmRoles(@BeanParam RequestParams requestParams, @PathParam("realm") String realm, @PathParam("userId") String userId);
-
-    @GET
-    @Path("userRoles")
-    @Produces(APPLICATION_JSON)
-    @Operation(operationId = "getCurrentUserRoles", summary = "Retrieve client roles for the currently authenticated user")
-    Role[] getCurrentUserRoles(@BeanParam RequestParams requestParams);
```

Comment:
Same in regards to this endpoint; don't we deprecate this, or introduce it as an API update?